welkin/models/document.py

```diff
@@ -0,0 +1,92 @@
+from io import BytesIO
+
+from welkin.models.base import Collection, Resource
+from welkin.models.util import find_patient_id_in_parents
+from welkin.pagination import PageableIterator
+
+
+class DocumentSummaryFile(Resource):
+    file: BytesIO
+
+    def get(self, patient_id: str = None, document_summary_id: str = None):
```

Comment:
I'd add a return annotation here since we're doing something other than returning `self`.

```suggestion
    def get(self, patient_id: str = None, document_summary_id: str = None) -> BytesIO:
```